rpc/src/rpc.rs

```diff
@@ -1726,12 +1728,12 @@ impl JsonRpcRequestProcessor {
                 "Invalid param: not a Token account".to_string(),
             ));
         }
-        let token_account = TokenAccount::unpack(account.data())
+        let token_account = StateWithExtensions::<TokenAccount>::unpack(account.data())
```

Comment:
Dang, sorry I didn't get back to these yet.

Reply:
No worries at all!